src/main/scala/vee/db/LevelDBWriter.scala

```diff
@@ -223,6 +242,9 @@ class LevelDBWriter(writableDB: DB, fs: FunctionalitySettings, val maxCacheSize:
               rw.delete(kTxIds)
               rw.put(kTxSeqNr, (txSeqNr - 1).max(0))
             }
+            val snapshot = rw.get(Keys.snapshot(addressId)(currentHeight))
```

Comment:
`addressId <- rw.get(Keys.changedAddresses(currentHeight))`
but not all `addressId` has `snapshot` in `currentHeight`